### ISSUE TITLE
Easy software disconnect

### ISSUE THREAD

**Issue by rustyrz** [cpebit/aawireless-issues#74]
I'd like to be able to use the phone occasionally without it launching AA. I realize I can unplug the power to the box, or shut off Bluetooth and Wi-Fi, but a software "toggle switch" would be very helpful. Maybe in the AA Wireless app?
Thank you!

**Comment by JackosXDA** [comment]
+1 good idea

**Comment by Snirpo** [comment]
We can do this, but only in start / stop mode.

**Comment by crowedavid** [comment]
@Snirpo any chance of this getting into a near term release?  i'd love to stop unplugging the device when i need to use a non-AA capable device but still have my phone with me.

**Comment by crowedavid** [comment]
perusing the other issues and it looks like #79  and #77 are also requesting the same functionality.

**Comment by Snirpo** [comment]
Can do, I'll add it to my list. You want a kind of pause functionality? What we can do is add a widget to turn on/off auto connect on the device. Is that an idea?

**Comment by rustyrz** [comment]
I think that makes sense; a simple way to tell it to leave the phone alone.
During most of my wife's driving she'd rather use the Honda dash stuff,
SiriusXM, etc. To toggle it at will would be great, kind of like plugging
in the cable but with a button.
Thanks!

**Comment by crowedavid** [comment]
> Can do, I'll add it to my list. You want a kind of pause functionality? What we can do is add a widget to turn on/off auto connect on the device. Is that an idea?

exactly.  widget would be a bonus. assuming this builds on the start/stop functionality, adding to the notification buttons on that there would work well too.

**Comment by rustyrz** [comment]
Yes, perfect. My wife will goes days and days without using it, so she is
disconnecting the USB. An easy on off button would be great.
Thank you!

**Comment by Snirpo** [comment]
Almost same functionality as #125 . We'll add this to app version 2.25.0.

**Comment by rustyrz** [comment]
Awesome, thank you!

**Comment by Snirpo** [comment]
We're postponing this a bit and bundling it with the next firmware (2.3.0). This firmware will also make it possible to control and change settings from another phone then the one currently connected for Android Auto.

**Comment by caco3** [comment]
I also miss an easy way to switch phones while both are in reach.
